support-frontend/assets/pages/new-contributions-landing/components/ContributionTypeTabs.jsx

```diff
@@ -55,11 +55,17 @@ const mapDispatchToProps = (dispatch: Function) => ({
 // ----- Render ----- //
 
 function ContributionTypeTabs(props: PropTypes) {
+  const contributionTypes = getValidContributionTypes(props.dropMonthlyVariant);
+
+  if (contributionTypes.length === 1 && contributionTypes[0] === 'ONE_OFF') {
```

Comment:
Would `if (contributionTypes === ['ONE_OFF'])` work?

Reply:
unfortunately no, you'd have to [implement equality check](https://stackoverflow.com/a/14853974)